es_sink/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Does this make this package dual licensed under both GPVv2 and MIT ? Is this intended ?